mass_mailing_partner/wizard/partner_mail_list_wizard.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+
+<record model="ir.ui.view" id="partner_mail_list_wizard_form">
```

Comment:
README usage instructions must explain how to get to this wizard and what does it do.